linetools/spectra/xspectrum1d.py

```diff
@@ -71,6 +80,51 @@ def from_spec1d(cls, spec1d):
         return slf
 
     @classmethod
+    def from_list(cls, xspecs, **kwargs):
+        """ Generate from a list of XSpectrum1D
+        By default, the excess pixels are masked
+        If sig or co is set in one, it should be set in all...
+        Units must be identical throughout too
+
+        Also note that masked pixels will be lost!
```

Comment:
Can you make this explanation more clear? 

Something like "Generate a single XSpectrum1D instance containing an array of spectra from a list of individual XSpectrum1D spectra. Each spectrum is padded with extra pixels so that the wavelength ranges of all spectra are covered. Padded pixels are masked." (?)